### PR TITLE
Cbrelease 3.0.2 rating review features

### PR DESCRIPTION
**Pull request by wilkysingh-tarento**
Rating Lookup and Summary Changes are done.

### CODE REVIEW

File: src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java [hunk 1]
```diff
@@ -147,6 +162,40 @@ public SBApiResponse upsertRating(RequestRating requestRating) {
...
+            } else {
+                String errMsg = "No ratings found for : " + lookupRequest.getActivity_Id() + ", activity_Type: " + lookupRequest.getActivity_Type();
+                response.put(Constants.MESSAGE, Constants.FAILED);
+                response.getParams().setErrmsg(errMsg);
+                response.setResponseCode(HttpStatus.BAD_REQUEST);
+            }
+        } catch (ValidationException ex) {
+            return processExceptionBody(response, ex, "", HttpStatus.BAD_REQUEST);
+        } catch (Exception e) {
+            System.out.println("The exception has occurred in lookup ratings" + e.getMessage());
```
Comment (wilkysingh-tarento) [inline src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java]: to be remove

Comment (wilkysingh-tarento) [reply src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java]: logging cbextlogger should be added

File: src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java [hunk 2]
```diff
@@ -69,28 +68,44 @@ public SBApiResponse getRatings(String activity_Id, String activity_Type, String
...
+            request.put(Constants.ACTIVITY_ID, activity_Id);
+            request.put(Constants.ACTIVITY_TYPE, activity_Type);
 
-    //TO DO
-    @Override
-    public SBApiResponse getRatingSummary(String activity_Id, String activity_Type) {
-        return null;
+            List<Map<String, Object>> existingDataList = cassandraOperation.getRecordsByProperties(Constants.KEYSPACE_SUNBIRD,
+                    Constants.TABLE_RATINGS_SUMMARY, request, null);
+            if (!existingDataList.isEmpty()) {
```
Comment (wilkysingh-tarento) [inline src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java]: Collectutil.isEmpty is more appropriate

File: src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java [hunk 3]
```diff
@@ -69,28 +68,44 @@ public SBApiResponse getRatings(String activity_Id, String activity_Type, String
...
+                response.put(Constants.MESSAGE, Constants.SUCCESSFUL);
+                response.put(Constants.RESPONSE, existingDataList);
+                response.setResponseCode(HttpStatus.OK);
+            } else {
+                String errMsg = "No reviews found for : " + activity_Id + ", activity_Type: " + activity_Type;
```
Comment (wilkysingh-tarento) [inline src/main/java/org/sunbird/ratings/service/RatingServiceImpl.java]: Dont hard code string

**Review by pkranga — APPROVED** [review]
Approved

**Pull request closed without merging**